api/src/routers/meals.js

```diff
@@ -0,0 +1,93 @@
+import "dotenv/config";
+import express from "express";
+import knex from "../database_client.js";
+
+const mealsRouter = express.Router();
+
+// Future Meals
+mealsRouter.get("/future-meals", async (req, res) => {
+  try {
+    const futureMealsQuery =
+      process.env.DB_CLIENT === "mysql2"
+        ? "SELECT * FROM meal WHERE `when` > NOW() ;"
+        : "SHOW TABLES;";
```

Comment:
```suggestion
    const futureMealsQuery = "SELECT * FROM meal WHERE `when` > NOW() ;";
```
You do not have to handle the case where the database client is one that you do not support. Also, there is no need to keep the `SHOW TABLES;` functionality.

Reply:
Thank you, removed.